src/utils.js

```diff
@@ -78,9 +78,18 @@ var regexEscape = str => str ? str.replace(/[.*+\-?^${}()|[\]\\]/g, '\\$&') : ''
 var readJSON = file => {
```

Comment:
I don't think one should modify the `readJSON()` function. This function is for reading all JSON files, and JSON files only. If you want to read embedded JSON data when importing, then you should change the import function.

Reply:
Yes, that's true.